cfgov/jinja2/v1/youth_employment_success/goals.html

```diff
@@ -0,0 +1,91 @@
+<p>Meet Mya and Frankie, they’ll help you think about your own long-term goal.</p>
+<div class="content-l block">  
+  <div class="content-l_col-1-2">
+    <img width=400 height=300>
+    <p class="h2 u-mt15">Mya plans to bike to work</p>
+    <p class="u-mt20">Mya decided to bike to work because there’s a bikeshare near her school. Bikeshares allow you to borrow a bike from different self-service bike stations around the city. Mya heads straight to work from class—some days she has time to walk, but usually she’s in a rush and biking is faster.</p>
+    <p class="h3 u-bold">Mya’s long-term goal</p>
+    <p>I want to buy an annual bikeshare membership since my school offers a special student rate and it’d be a better deal than paying per ride. The $60 annual membership works out to a little over $1 per week. In order to save money for this, I will walk to work two days a week for the next few months.</p>
+  </div>
+  <div class="content-l_col-1-2">
+    <img width=400 height=300>
+    <p class="h2 u-mt15">Frankie plans to take the bus</p>
+    <p class="u-mt20">Frankie doesn’t live near public transportation so he usually gets a ride to the closest bus stop. The bus ride is about 25 minutes long so he leaves his house at 7:10am to allow enough time to get dropped off, ride the bus, and arrive at work before his 8am shift starts.</p>
+    <p class="h3 u-bold">Frankie’s long-term goal</p>
+    <p>I will take the bus five days a week, but over the next year, I’d like to save up enough money to buy a car so I don’t have to rely on friends and family for rides to the bus stop. In the meantime, I will buy a $50 monthly bus pass because it’s more cost-effective than paying $2 each way for a bus ticket.</p>
+  </div>
+</div>
+<p class="lead-paragraph">
+  Now let’s think through your long-term goal. Figuring out your long-term goal may help you figure out the best way to get to work now while you’re working towards this goal. 
+</p>
+<form class="block u-mt35">
+  <div class="block__sub">
+    {{
+      renderTextarea ({
+        'helperText': 'For example, saving up for an annual bus pass in the next few months or saving up to buy a car in the next year.',
+        'label': "What’s your long-term goal?",
+        'labelClass': "h3 u-bold",
+        'required': true
+      })
+    }}
+  </div>
+  <div class="block__sub">
+    {{
+      renderTextarea ({
+        'helperText': 'This goal could help you save money, in the case of an annual bus pass, or maybe it’s a more reliable option, such as having your own car instead of relying on rides.',
+        'label': "Why is this goal important to you?",
+        'labelClass': "h3 u-bold",
+        'required': true
+      })
+    }}
+  </div>
+  <div class="block__sub">
+    <span class="h3 u-bold u-mb0">
+      When do you hope to reach your goal? 
+    </span>
+    <p class="small u-mt0">
```

Comment:
Can this use [`<small>` markup](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/small), or `u-small` or `a-micro-copy`? (See https://github.com/cfpb/capital-framework/issues/962)